JSON-connect/.clasp.json

```diff
@@ -1 +1 @@
-{"scriptId":"1y2Cax2-fmQ_Iea9RIG4kc_wTfLevkV3KW4cYvNIxMk9p5qdVpXmnpnU9","rootDir":"src"}
+{"scriptId":"1JVKGb63vmuX7KJUjWKVpOds3rq550FLeL5ZGjBtTb2sAhvzDeJdLRxt7","rootDir":"src"}
```

Comment:
Please revert this change, this is the script id that we use for the deployment.

Reply:
done